Cargo.toml

```diff
@@ -16,7 +16,7 @@ text-size.opt-level = 3
 miniz_oxide.opt-level = 3
 
 [profile.release]
-incremental = true
+incremental = true 
```

Comment:
trailing ws?

Reply:
Oh, for some reason whatever formats that in vscode for me adds that in and I'm not sure why 😅

Reply:
One of these days, I'll make a proper lossless toml library...

Reply:
There actually is a lossless toml library in rust(it even uses rowan) https://github.com/tamasfe/taplo and I just noticed I use the vscode extension of that but it seems the formatter for that doesn't play nicely yet.

Reply:
Strange, Even Better TOML works fine for me (with some consistent config tweaking).